src/components/UserForm.tsx

```diff
@@ -11,12 +11,13 @@ const UserForm = () => {
   const [year, setYear] = useState(0);
   const [kilometres, setKilometres] = useState(0);
   const [price, setPrice] = useState(0);
+  const [downpayment, setDownPayment] = useState(0);
```

Comment:
If we're going to stick to having downpayment as one word in our internal implementation, we should probably make this `setDownpayment`. I wouldn't be against using `downPayment` everywhere, that would require a lot of changes though, and the constent string for the Senso API does have to be `downpayment`, your call though.

Reply:
Yea, good point